pypeit/tests/test_load.py

```diff
@@ -50,7 +50,7 @@ def test_load_1dspec():
     assert isinstance(spec, XSpectrum1D)
     # By objname
     #spec2 = load.load_1dspec(spec_file, objname='O473-S5473-D01-I0008')
-    spec2 = load.load_1dspec(spec_file, objname='SPAT0132-SLIT0001-DET01-SCI008')
+    spec2 = load.load_1dspec(spec_file, objname='SPAT0132-SLIT0000-DET01')
```

Comment:
Should this really be slit=0000?
I thought @jhennawi had set slits to start counting at 1.

Reply:
No all slits are counting from zero.